### PR TITLE
Final copy

### PR DESCRIPTION
**Pull request by rossbruniges**
Working on the files supplied by Matt at part of the work to complete #8

### CODE REVIEW

File: gameon/static_site/templates/static_site/faqs.html [hunk 1]
```diff
@@ -60,7 +75,7 @@ <h2 class="whimper">Index</h2>
                     <li><a href="#game-jams">Whats up with the Game Jams?</a></li>
                     <li><a href="#help">I need help. Who do I contact?</a></li>
                     <li><a href="#firefox-os">What is Firefox OS?</a></li>
-                    <li><a href="#mozilla-marketplace">What is the Mozilla Marketplace?</a></li>
+                    <li><a href="#mozilla-marketplace">What is the Firefox Marketplace?</a></li>
```
Comment (andrewhayward) [inline gameon/static_site/templates/static_site/faqs.html]: I have a feeling this should link to `#firefox-marketplace`.

**Comment by andrewhayward** [comment]
This might be pushing the boundaries of anality a little, but you've got one or two `’`s in amongst your `'`s. Are we fussed about this little inconsistency?

**Comment by rossbruniges** [comment]
Have fixed the punctuation - the joys of cutting and pasting from etherpads!

File: gameon/static_site/templates/static_site/faqs.html (hunk 1 above)
Comment (rossbruniges) [inline gameon/static_site/templates/static_site/faqs.html]: Yes - now wondering if it's supposed to be the Firefox or Mozilla Marketplace...

**Pull request merged**